### DISCUSSION TITLE
Queations

### DISCUSSION THREAD

**Discussion by khurlvergara** [khurlvergara/Tafe22S207#1] [General]
Tafadzwa I will create the conversion features do you have any idea about it?

**Comment by Tafadzwa10** [comment]
Ok I will work on the Mortgage Calculator, You can Apply some try-catch blocks and see how that goes.

**Comment by khurlvergara** [comment]
In creating the branch what should we do?

**Reply by Tafadzwa10** [reply]
You should checkout of the development branch and then create your conversion_feature 2

**Comment by Tafadzwa10** [comment]
When merging the two branches which branch should we checkout?

**Reply by khurlvergara** [reply]
the development branch that's the branch that will begin to merge

**Comment by Tafadzwa10** [comment]
Should we push all branches after the merge or just the development branch?

**Reply by khurlvergara** [reply]
before the merging, we already push the feature branches so after that we just need to push the development branch

**Comment by khurlvergara** [comment]
Does the code work well and has nothing to debug?

**Reply by Tafadzwa10** [reply]
Yes, the code is functioning properly and doesn't require further debugging.

**Comment by Tafadzwa10** [comment]
How did you go with debugging the conversion_feature 2?

**Reply by khurlvergara** [reply]
I used the debugging tool on the visual studio to see the problem of it and try to figure out the solution.
